R/process.R

```diff
@@ -1166,6 +1169,7 @@ process_parse_data <- function(data, dataset_id, metadata, contexts) {
   }
 
   # Implement any value changes as per substitutions
+  # I think this breaks if substitutions is .na in the metadata?
   if (!is.na(metadata[["substitutions"]][1])) {
```

Comment:
Please check if it breaks. the format `is.na(metadata[["substitutions"]][1]`is intended to catch instances where `substitutions: .na` in the yaml

Reply:
All good, it does not break. It was only breaking for Canham_2023 because `read_yaml` wasn't able to read a special character (Ψ) so it was never able to read substitutions (hence substitutions was NULL instead of NA).